flake.nix

```diff
@@ -44,6 +52,7 @@
           myModifier = drv:
             pkgs.haskell.lib.addBuildTools drv (with hsPkgs; [
               cabal-install
+              pythonDevEnv
```

Comment:
in my local flake, I just added pkgs.jupyter to test the installation of kernels. If you want advanced testing facilities, you can run the flake at https://github.com/tweag/jupyterWith/ and use `--override-input ihaskell ~/myihaskell` for instance.

Reply:
Hmm, it sounds like it would be good to include this by default. Would that cause any issues for you?

Reply:
I was planning to submit a PR with that, but I dont know ifit's enough for OP's purpose.

Reply:
I don't have much of a preference here. It sounds like a good idea :+1: